test/v2/models/request/submitBsas/foreignProperty/ForeignPropertyExpensesSpec.scala

```diff
@@ -31,13 +31,38 @@ class ForeignPropertyExpensesSpec extends UnitSpec {
       |  "travelCosts": 123.12,
       |  "costOfServices": 123.12,
       |  "residentialFinancialCost": 123.12,
-      |  "other": 123.12
+      |  "other": 123.12,
+      |  "consolidatedExpenses": 123.12
+      |}
+      |""".stripMargin)
+
+  val desJson = Json.parse(
+    """
+      |{
+      |  "premisesRunningCosts": 123.12,
+      |  "repairsAndMaintenance": 123.12,
+      |  "financialCosts": 123.12,
+      |  "professionalFees": 123.12,
+      |  "travelCosts": 123.12,
+      |  "costOfServices": 123.12,
+      |  "residentialFinancialCost": 123.12,
+      |  "other": 123.12,
+      |  "consolidatedExpense": 123.12
```

Comment:
Use different numbers for all of the fields. Using the same number will not catch ordering problems with the Json Writes.